### PR TITLE
fix: add display name in qr profile

### PR DESCRIPTION
**Pull request by clegirar**
Fix https://github.com/orgs/berty/projects/42#card-74281369

Signed-off-by: clegirar <clemntgirard@gmail.com>

### CODE REVIEW

File: js/packages/components/settings/Home.tsx [hunk 1]
```diff
@@ -85,9 +85,27 @@ const HomeHeaderAvatar: React.FC<{ navigation: ComponentProps<typeof Home>['navi
...
+					displayName: account.displayName,
+				})
+				if (ret?.internalUrl) {
+					setLink(ret?.internalUrl)
+				}
+			}
+		}
+		getAccountLink().then()
+		// eslint-disable-next-line react-hooks/exhaustive-deps
+	}, [])
```
Comment (n0izn0iz) [inline js/packages/components/settings/Home.tsx:108]: this will make the call on every render

File: js/packages/components/settings/Home.tsx [hunk 2]
```diff
@@ -85,9 +85,27 @@ const HomeHeaderAvatar: React.FC<{ navigation: ComponentProps<typeof Home>['navi
...
+				const ret = await ctx.client?.instanceShareableBertyID({
+					reset: false,
+					displayName: account.displayName,
+				})
+				if (ret?.internalUrl) {
+					setLink(ret?.internalUrl)
+				}
+			}
+		}
+		getAccountLink().then()
```
Comment (n0izn0iz) [inline js/packages/components/settings/Home.tsx:106]: `.then()` is not needed

File: js/packages/components/settings/Home.tsx [hunk 3]
```diff
@@ -108,10 +126,10 @@ const HomeHeaderAvatar: React.FC<{ navigation: ComponentProps<typeof Home>['navi
 						{account?.displayName || ''}
 					</Text>
 					<View style={[padding.top.scale(18 * scaleHeight)]}>
-						{(account?.link && (
+						{(link && (
```
Comment (n0izn0iz) [inline js/packages/components/settings/Home.tsx:129]: maybe display a spinner when `!link`

**Comment by zxxma** [comment]
iOS Reviewed : LGTM 👍

**Pull request merged**